app/converters/shopify/variant_converter.rb

```diff
@@ -0,0 +1,35 @@
+module Shopify
+  class VariantConverter
+    def initialize(spree_variant)
+      @spree_variant = spree_variant
+    end
+
+    def to_hash
+      {
```

Comment:
Should this be collapsed into a private method that is called by the `to_hash` interface?